### PR TITLE
Refactoring & Enhanced Code Readability

### PR DESCRIPTION
**Pull request by nulledphx**

### CODE REVIEW

**Review by nulledphx — COMMENTED** [review]
👍📌

File: src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java [hunk 1]
```diff
@@ -13,9 +18,9 @@
 import com.velocitypowered.api.proxy.ProxyServer;
 import com.velocitypowered.api.proxy.server.RegisteredServer;
 import com.velocitypowered.api.proxy.server.ServerInfo;
-import moe.yunfachi.yunfaremember.commands.YunfaRememberCommand;
-import moe.yunfachi.yunfaremember.config.Players;
-import moe.yunfachi.yunfaremember.config.Settings;
+import moe.yunfachi.yunfaremember.commands.COMAND;
+import moe.yunfachi.yunfaremember.config.mewnjHHHHH;
```
Comment (BANanaD3V) [inline src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java:22]: Please change to something normal

File: src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java [hunk 2]
```diff
@@ -13,9 +18,9 @@
...
+import moe.yunfachi.yunfaremember.config.jfjfiSNdh;
```
Comment (BANanaD3V) [inline src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java:23]: Change to something normal too

File: src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java [hunk 3]
```diff
@@ -31,236 +36,152 @@
...
+import java.util.zip.ZipEntry;
 
-@Plugin(
-        id = "yunfaremember",
-        name = "yunfaRemember",
-        version = "1.1.1",
-        description = "A velocity plugin allows you to stay on the same server when you exit",
-        url = "https://modrinth.com/plugin/yunfaremember",
-        authors = {"yunfachi"}
-)
```
Comment (BANanaD3V) [inline src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java:43]: Fix formatting as it was before

File: src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java [hunk 4]
```diff
@@ -31,236 +36,152 @@
...
-    @Inject
-    private Metrics.Factory metricsFactory;
-    private static final int METRICS_ID = 19423;
-    private static final int SPIGOT_RESOURCE_ID = 111843;
-    private final ProxyServer server;
-    private final Logger logger;
-    private final Path dataDirectory;
-    private Settings settings;
-    private Players players;
+    public YunfaRemember(@NotNull ProxyServer e, @NotNull Logger f, @DataDirectory Path g) {
```
Comment (BANanaD3V) [inline src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java:57]: We should change letters to actual names

File: src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java [hunk 5]
```diff
@@ -31,236 +36,152 @@
...
-                                )
-                        );
-                        getConfig().getServerGroups().forEach((k, v) -> {
-                            if(v.contains(server.get().getServerInfo().getName())) {
-                                if(players.getLatestServer(event.getPlayer().getUniqueId(),k).equals(server.get().getServerInfo().getName())) {
-                                    players.setLatestServer(
-                                            event.getPlayer().getUniqueId(),
-                                            k,
-                                            server.get().getServerInfo().getName()
+    public void t     (@NotNull ServerPreConnectEvent u) {
```
Comment (BANanaD3V) [inline src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java:77]: This entire function contains letters and strange naming

File: src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java [hunk 6]
```diff
@@ -31,236 +36,152 @@
...
-                        event.setInitialServer(server.get());
-                        if(!settings.getServerGroups().containsKey(server.get().getServerInfo().getName()))
-                            break;
-                        getConfig().getServerGroups().forEach((k, v) -> {
-                            if(v.contains(server.get().getServerInfo().getName())) {
-                                players.setLatestServer(
-                                        event.getPlayer().getUniqueId(),
-                                        k,
-                                        server.get().getServerInfo().getName()
+    public void ee(@NotNull PlayerChooseInitialServerEvent ff) {
```
Comment (BANanaD3V) [inline src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java:113]: Strange naming

File: src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java [hunk 7]
```diff
@@ -31,236 +36,152 @@
...
-            result = event.getInitialServer().get();
-        }
-        RegisteredServer finalResult = result;
-        settings.getServerGroups().forEach((k, v) -> {
-            if(v.contains(finalResult.getServerInfo().getName())) {
-                players.setLatestServer(
-                        event.getPlayer().getUniqueId(),
-                        k,
-                        finalResult.getServerInfo().getName()
+            } else {jj(ff, null); /* devnull */}
```
Comment (BANanaD3V) [inline src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java:130]: Same here

File: src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java [hunk 8]
```diff
@@ -272,62 +193,80 @@ public UpdateChecker getUpdateChecker() {
...
+
+    private void s() {
+        if (h.hjkl()) {
+            getUpdateChecker().check().thenAccept(yy -> {
+                if (!yy.isUpToDate()) {log(Level.WARN, "A new version of yunfaRemember is available: v" + yy.getLatestVersion() + " (running v" + getVersion() + ")\nDownload: https://modrinth.com/mod/yunfaremember");}
             });
         }
     }
 
-    public void log(@NotNull Level level, @NotNull String message, @NotNull Throwable... exceptions) {
```
Comment (BANanaD3V) [inline src/main/java/moe/yunfachi/yunfaremember/YunfaRemember.java:286]: Please, change it

File: src/main/java/moe/yunfachi/yunfaremember/config/mewnjHHHHH.java [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+                    ┌─────────────────────────────────────────────────────┐
+                    │                yunfaRemember Players                │
+                    ├─────────────────────────────────────────────────────┤
+                    │ Author: yunfachi                                    │
+                    │ Site: https://yunfachi.dev                          │
+                    │ Github: https://github.com/yunfachi/yunfaRemember   │
+                    │ Modrinth: https://modrinth.com/plugin/yunfaremember │
+                    │ Spigot: https://www.spigotmc.org/resources/111843   │
+                    └─────────────────────────────────────────────────────┘""")
+public class mewnjHHHHH {
```
Comment (BANanaD3V) [inline src/main/java/moe/yunfachi/yunfaremember/config/mewnjHHHHH.java:38]: Bad naming everywhere in file

File: src/main/java/moe/yunfachi/yunfaremember/config/jfjfiSNdh.java [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+        ┌─────────────────────────────────────────────────────┐
+        │                yunfaRemember Config                 │
+        ├─────────────────────────────────────────────────────┤
+        │ Author: yunfachi                                    │
+        │ Site: https://yunfachi.dev                          │
+        │ Github: https://github.com/yunfachi/yunfaRemember   │
+        │ Modrinth: https://modrinth.com/plugin/yunfaremember │
+        │ Spigot: https://www.spigotmc.org/resources/111843   │
+        └─────────────────────────────────────────────────────┘""")
+public class jfjfiSNdh {
```
Comment (BANanaD3V) [inline src/main/java/moe/yunfachi/yunfaremember/config/jfjfiSNdh.java:23]: Bad naming in file

**Review by BANanaD3V — CHANGES_REQUESTED** [review]
Please change naming to something normal

**Pull request closed without merging**